server/source/server.ts

```diff
@@ -81,6 +83,20 @@ router.get('/oauth', async (ctx) => {
 	}
 })
 
+router.post('/send-crisp-message', koaBody(), async (ctx) => {
+	try {
+		const body = ctx.request.body as unknown as BodyType
```

Comment:
C'est peut être un peu parano mais j'aurais bien ajouter une fonction qui parse ton body pour checker que c'est bien des string ou autre comme input au cas ou on t'envoie n'importe quoi, et typer en même temps
```suggestion
		const body = parseCrsipBody(ctx.request.body)
```

Reply:
Tu penses qu'il faudrait parser après le parsing opéré par `koa-body` ? Tu verrais quelles validations à ajouter ?

Reply:
bah koa-body transforme juste le body full texte en objet/array/string/etc., il sais pas comment tu t'en sert après, il faudrait check que tu as bien les var `message`, `email` et `subject` j'imagine, et que c'est bien des string

Reply:
Okay j'ai ajouté un fonction utilitaire de validation, j'ai aussi limité le nombre de caractères à 254 pour les emails (limite max d'une adresse email d'après mes recherches, en vrai j'ai trouvé plusieurs réponses qui divergeaient et j'ai pris la valeur basse)